### PR TITLE
[INTEL MKL] Reverting changes made to Add + Conv fusion during MKL-DNN v1.x integration

### PR DESCRIPTION
**Pull request by bhavani-subramanian**
This fixes the accuracy regression in Resnet-50 for FP32.

### CODE REVIEW

File: tensorflow/core/kernels/mkl_conv_ops.cc [hunk 1]
```diff
@@ -972,40 +970,34 @@ class MklConvOp : public OpKernel {
...
     if (fuse_add_) {
       const Tensor& add_tensor = MklGetInput(context, kInputIndex_Add);
       MklDnnShape add_mkl_shape;
       GetMklShape(context, kInputIndex_Add, &add_mkl_shape);
 
-      // Check if reorder is needed
+      // Check if need reorder
       if (add_mkl_shape == output_mkl_shape) {
-        DCHECK((*output_tensor)->CopyFrom(add_tensor, output_tf_shape));
+        CHECK((*output_tensor)->CopyFrom(add_tensor, output_tf_shape));
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_conv_ops.cc]: We don't allow new `CHECK`s in the code anymore. Please use
```c++
auto result = (*output_tensor)->CopyFrom(add_tensor, output_tf_shape);
DCHECK(result);
```
or something.

**Review by penpornk — CHANGES_REQUESTED** [review]
One change please.

File: tensorflow/core/kernels/mkl_conv_ops.cc (hunk 1 above)
Comment (bhavani-subramanian) [reply tensorflow/core/kernels/mkl_conv_ops.cc]: @penpornk Thanks for the quick review! I have changed `CHECK` to `DCHECK`.

File: tensorflow/core/kernels/mkl_conv_ops.cc [hunk 2]
```diff
@@ -979,7 +979,7 @@ class MklConvOp : public OpKernel {
 
       // Check if need reorder
       if (add_mkl_shape == output_mkl_shape) {
-        CHECK((*output_tensor)->CopyFrom(add_tensor, output_tf_shape));
+        DCHECK((*output_tensor)->CopyFrom(add_tensor, output_tf_shape));
```
Comment (penpornk) [inline tensorflow/core/kernels/mkl_conv_ops.cc]: DCHECK gets optimized out when TF is not in debug mode. Any statements inside DCHECK will not be executed in opt mode. It looks like you need `CopyFrom` to be executed so you should call `CopyFrom` in one line, save the result, and `DCHECK` the result in the next line.

If `CopyFrom` doesn't actually need to be run, please let me know and I'll approve the code.

**Review by penpornk — CHANGES_REQUESTED** [review]

**Comment by bhavani-subramanian** [comment]
@penpornk Done.

**Review by penpornk — APPROVED** [review]
Thank you for your quick fixes!

**Pull request merged**